sway-save-tree/sway-save-tree

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import re
+import subprocess
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-s', '--socket', help='Socket')
+parser.add_argument('-w', '--workspace',
+        help='Specifies the workspace that should be dumped, e.g. 1. ' \
+                'This can either be a name or the number of a workspace.')
+parser.add_argument('-o', '--output',
+        help='Specifies the output that should be dumped, e.g. LVDS-1')
+parser.add_argument('-v', '--version', action='store_true', help='Version')
+args = parser.parse_args()
+
+if args.version:
+    print('sway-save-tree version %s' % '1.0.0')
+    exit(0)
+
+cmd = ['swaymsg', '-t', 'get_tree']
```

Comment:
You could also use Python sockets to perform IPC, though it would take about a dozen extra lines.

Also, this should all really be in a `main` function.